src/apps/beefy/shortcuts.ts

```diff
@@ -0,0 +1,121 @@
+import { Address, parseUnits, erc20Abi, encodeFunctionData } from 'viem'
+import { z } from 'zod'
+import { logger } from '../../log'
+import { getClient } from '../../runtime/client'
+import {
+  simulateTransactions,
+  UnsupportedSimulateRequest,
+} from '../../runtime/simulateTransactions'
+import { ZodAddressLowerCased } from '../../types/address'
+import { NetworkId } from '../../types/networkId'
+import {
+  ShortcutsHook,
+  createShortcut,
+  tokenAmounts,
+  Transaction,
+} from '../../types/shortcuts'
+import { poolAbi } from './abis/pool'
+
+// Hardcoded fallback if simulation isn't enabled
+const DEFAULT_DEPOSIT_GAS = 500_000n
```

Comment:
have you validated that this gas is sufficient in case the simulation doesn't work? from a quick look at the tx you linked on the PR desc, it looks like it used more than this

Reply:
Looked at a few different deposit transactions, all looked to use around 600,000-650,000 so set it at 750,000